### ISSUE TITLE
Endless loop when a class has an attribute called `private`

### ISSUE THREAD

**Issue by wgc-as** [bbatsov/rubocop#1563]
Rubocop falls into an endless loop with this class:

```ruby
class Test
  def test_method
    if private
      puts 'private'
    end
  end
end
```

It adds an empty line before `if private` then removes it, then adds it back...

We have a class which has an attribute called `private`, so this is painful for us. Please fix it.

**Comment by bbatsov** [comment]
What version of RuboCop are you using?

**Comment by wgc-as** [comment]
Version 0.28.0. I'm on Mac OSX Mavericks.
Here is how to reproduce:
```
⌁ [user:~/tmp] $ cd ~/tmp
⌁ [user:~/tmp] $ cat > test.rb
class Test
  def test_method
    if private
      puts 'private'
    end
  end
end
⌁ [user:~/tmp] 4s $ gem install rubocop
Fetching: rainbow-2.0.0.gem (100%)
Successfully installed rainbow-2.0.0
Fetching: ast-2.0.0.gem (100%)
Successfully installed ast-2.0.0
Fetching: parser-2.2.0.2.gem (100%)
Successfully installed parser-2.2.0.2
Fetching: powerpack-0.0.9.gem (100%)
Successfully installed powerpack-0.0.9
Fetching: astrolabe-1.3.0.gem (100%)
Successfully installed astrolabe-1.3.0
Fetching: ruby-progressbar-1.7.1.gem (100%)
Successfully installed ruby-progressbar-1.7.1
Fetching: rubocop-0.28.0.gem (100%)
Successfully installed rubocop-0.28.0
7 gems installed
⌁ [user:~/tmp] 40s $ rubocop test.rb
warning: parser/current is loading parser/ruby21, which recognizes
warning: 2.1.5-compliant syntax, but you are running 2.1.2.
Inspecting 1 file
C

Offenses:

test.rb:1:1: C: Missing top-level class documentation comment.
class Test
^^^^^
test.rb:3:5: C: Use a guard clause instead of wrapping the code inside a conditional expression.
    if private
    ^^
test.rb:3:5: C: Favor modifier if usage when having a single-line body. Another good alternative is the usage of control flow &&/||.
    if private
    ^^
test.rb:3:8: C: Keep a blank line before and after private.
    if private
       ^^^^^^^

1 file inspected, 4 offenses detected
⌁ [user:~/tmp] 1 $ rubocop -a test.rb
warning: parser/current is loading parser/ruby21, which recognizes
warning: 2.1.5-compliant syntax, but you are running 2.1.2.
Inspecting 1 file
^C
Exiting... Interrupt again to exit immediately.
^C⌁ [user:~/tmp] 10s 1 $
```

**Comment by bquorning** [comment]
This is a duplicate of #1484, which was fixed on master by #1491.

**Comment by wgc-as** [comment]
Indeed it is fixed in master. Any plans to release a new version? There are a bunch of new commits in the master branch since the last release. Thanks.
